app/routes/organen/orgaan/mandatarissen.js

```diff
@@ -23,10 +24,71 @@ export default class OrganenMandatarissenRoute extends Route {
     }
 
     return {
-      mandatarissen,
+      mandatarissen: await this.foldMandatarissen(mandatarissen),
     };
   }
 
+  async foldMandatarissen(mandatarissen) {
+    const personMandaatData = {};
+    const folded = [];
+    await Promise.all(
+      mandatarissen.map(async (mandataris) => {
+        const personId = (await mandataris.isBestuurlijkeAliasVan).id;
+        const mandaatId = (await mandataris.bekleedt).id;
+        const key = `${personId}-${mandaatId}`;
+        const existing = personMandaatData[key];
+        if (existing) {
+          existing.foldedStart = moment.min(
+            moment(existing.foldedStart),
+            moment(mandataris.start)
+          );
+          if (!mandataris.einde || !existing.foldedEnd) {
```

Comment:
Not sure why this is here for end and not for start. Probably we want to require a start, but to be completely save, I would not presume this here.